pkg/webhook/pod/mutating/pod_create_update_handler.go

```diff
@@ -73,6 +73,11 @@ func (h *PodCreateHandler) Handle(ctx context.Context, req admission.Request) ad
 		return admission.Errored(http.StatusInternalServerError, err)
 	}
 
+	err = h.containerLaunchPriorityInitialization(ctx, req, obj)
```

Comment:
plz comments that "the order matters and sidecarsetMutatingPod must precede containerLaunchPriorityInitialization"